benchmarks/time_to_interact.py

```
@@ -95,11 +108,15 @@ def run_benchmark(client: Celesto, cpus: int, memory: int, run_index: int, verbo
     results["exec_min"] = round(min(exec_times), 3)
     results["exec_max"] = round(max(exec_times), 3)
     if verbose:
-        print(f"  Warm exec (5x): avg={results['exec_avg']:.3f}s p50={results['exec_p50']:.3f}s min={results['exec_min']:.3f}s max={results['exec_max']:.3f}s")
+        print(
+            f"  Warm exec (5x): avg={results['exec_avg']:.3f}s p50={results['exec_p50']:.3f}s min={results['exec_min']:.3f}s max={results['exec_max']:.3f}s"
+        )
 
     # 5. Complex exec (real workload)
     complex_time, complex_result = measure(
```

Ruff (0.15.20)

[warning] Unpacked variable `complex_result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)